### ISSUE TITLE
Not able to add a user to my DAX Optimizer account/group.

### ISSUE THREAD

**Issue by dpesante** [tabulartools/dax-optimizer#43]
### Describe the problem

I have an account with valid licensese for 5 datamodels with 20 runs per day. My account is under davidp@pesanteanalytics.com. 

I am trying to add other members of my pesanteanalytics.com team but when I try to add the user to my group, I search the user by email and it is not allowing me to find them nor gives me an option to send them an invite to join. So, one of my members... sandrae@pesanteanalytics.com created an account (no licenses) and still I can not find her. What is the correct approach to adding my other team members to my group so that we can share the licenses that I have?

### Power BI

Latest Version

**Comment by marcosqlbi** [comment]
Did you create accounts using Microsoft Entra ID work account, or by defining a password for the authentication in DAX Optimizer?
Group accounts can be created only with work accounts. The guide to create and manage a group is here:
https://docs.daxoptimizer.com/how-to-guides/managing-groups

However, if you assigned the license to an individual user and you want to assign it to a group, we must manually reissue the license - in such a case, please write to support@tabulartools.com so we can assist for that.

I look forward to hearing from you.

**Comment by albertospelta** [comment]
As Marco mentioned, groups are a feature available only for DAX Optimizer accounts authenticated using a Microsoft Work Account (Entra ID). In this case, Sandra’s account has been created using a username/password, which is why you are unable to find it. Sandra should create a new DAX Optimizer account by selecting "Microsoft Work Account" during registration. Once the new account is created, it should be possible to add it to your group.

**Comment by dpesante** [comment]
Yes.. my account was created using the Microsoft… the other account was created with password… So, let me ask another team member to create an account but using MF authentication instead of password to see if I can see it from my account when I add another user….

Thanks!

**Comment by dpesante** [comment]
Here is my current license under my ***@***.******@***.***> account (MF Aunthentication account)

**Comment by albertospelta** [comment]
@dpesante I just checked, and I see that the users have been added to your group. Can you confirm if the issue with adding them has been resolved? 

Regarding the license, if you need to transfer it from your account to the group, please write to support@tabulartools.com and include the license key you wish to transfer. You can find the license key in the `Settings > Licenses` panel. 
For more information, see https://docs.daxoptimizer.com/how-to-guides/managing-licenses.

Thanks!

**Comment by dpesante** [comment]
I still need help with this…

https://app.screencast.com/69hrQsE6gGjhs

thanks!

From: David Pesante
Sent: Thursday, November 28, 2024 4:13 PM
To: tabulartools/dax-optimizer ***@***.***>; tabulartools/dax-optimizer ***@***.***>
Cc: Author ***@***.***>
Subject: RE: [tabulartools/dax-optimizer] Not able to add a user to my DAX Optimizer account/group. (Issue #43)

Here is my current license under my ***@***.******@***.***> account (MF Aunthentication account)


***@***.***

**Comment by marcosqlbi** [comment]
I appreciate the detailed request.
First, the token you mentioned is only for the command-line access to automate the analysis. However, this is not related to your current issue.
You can find all the details here: https://docs.daxoptimizer.com/how-to-guides/automate-analysis

You correctly created a group with your colleagues. However, if you assigned the license to your personal account, you have to reassign the license to the group. To do that, please write to [support@tabulartools.com](mailto:support@tabulartools.com) and include the license key you wish to transfer (the one you tried to copy/paste in the video). We will deactivate that license and provide a new redemption code that you will use to activate the license on the group.

Thanks!

**Comment by dpesante** [comment]
Great … thanks!

**Comment by marcosqlbi** [comment]
@dpesante we replied to your email but have not received a follow-up. Please confirm that you received our replies and that you solved the issue.

**Comment by dpesante** [comment]
Sorry..I thought I replied back...

Yes, we are good to go and it is working very well...

Thanks!!!

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
